GoogleTestAdapter/Core/GoogleTestConstants.cs

```diff
@@ -18,6 +18,8 @@ public static class GoogleTestConstants
         public const string NrOfRepetitionsOption = " --gtest_repeat";
         public const string CatchExceptions = " --gtest_catch_exceptions";
         public const string BreakOnFailure = " --gtest_break_on_failure";
+        public const string PrintUTF8 = " --gtest_print_utf8=1";
```

Comment:
Are we using this one anywhere? I see the XMLOutput being used in the baseCommandLine below but dont see other references to this flag.

Reply:
We are not, added it as a constant just in case for the future